### PR TITLE
CONTROLLERS : Get All `Milestone`

### LINKED ISSUE #319: CONTROLLERS: Get All `Milestone`
Don't forget to add OData

### PR DESCRIPTION
**Pull request by ShohruhUzDev**
Closes #319

### CODE REVIEW

**Review by ElbekDeveloper — APPROVED** [review]

File: Tarteeb.Api/Controllers/MilestonesController.cs [hunk 1]
```diff
@@ -51,15 +53,35 @@ public async ValueTask<ActionResult<Milestone>> PostMilestoneAsync(Milestone mil
...
         [HttpPut]
         public async ValueTask<ActionResult<Milestone>> PutMilestoneAsync(Milestone milestone)
         {
             try
             {
                 Milestone modifiedMilestone =
                     await this.milestoneServices.ModifyMilestoneAsync(milestone);
-
                 return Ok(modifiedMilestone);
+
```
Comment (ElbekDeveloper) [inline Tarteeb.Api/Controllers/MilestonesController.cs:84]: line

File: Tarteeb.Api/Controllers/MilestonesController.cs [hunk 2]
```diff
@@ -74,14 +96,7 @@ public async ValueTask<ActionResult<Milestone>> PutMilestoneAsync(Milestone mile
...
-                return InternalServerError(milestoneDependencyException.InnerException);
-            }
-            catch (MilestoneServiceException milestoneServiceException)
-            {
-                return InternalServerError(milestoneServiceException.InnerException);
-            }
         }
     }
 }
+
```
Comment (ElbekDeveloper) [inline Tarteeb.Api/Controllers/MilestonesController.cs:102]: Nimaga PutMilestone metodigangi catch bloklar o'chirib tashlandi?

**Review by ElbekDeveloper — CHANGES_REQUESTED** [review]

File: Tarteeb.Api/Controllers/MilestonesController.cs (hunk 2 above)
Comment (Jahongir1507) [reply Tarteeb.Api/Controllers/MilestonesController.cs:101]: Uchirib tepaga utkazdim, githubni uzida conflictni tug'irlagandim

File: Tarteeb.Api/Controllers/MilestonesController.cs (hunk 1 above)
Comment (Jahongir1507) [reply Tarteeb.Api/Controllers/MilestonesController.cs]: resolved

**Pull request merged**